src/algebra/star/chsh.lean

```diff
@@ -162,28 +162,6 @@ local notation `√2` := (real.sqrt 2 : ℝ)
 
 namespace tsirelson_inequality
 
-/-!
-We next need some lemmas about numerals in modules and algebras.
-The awkward appearance of both `•ℤ` and `•` seems unavoidable because later calculations by `abel`
-will introduce `•ℤ`.
-If anyone sees how to obtain these from general statements, please improve this!
--/
```

Comment:
Oh, just being able to delete this with no other changes in the file makes me so happy. :-)